set.mm

```diff
@@ -611927,6 +611927,35 @@ fixed reference functional determined by this vector (corresponding to
 #*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
 $)
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Russell's paradox
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d x y z A $.
+    $( Russell's Paradox, generalized to elementhood in all classes.  See ~ ru
+       for the non-generalized version.
```

Comment:
I would propose:
> Relativized version of Russell's paradox ~ ru (which corresponds to ` A = _V ` ).

Reply:
I know it's probably simply "relative to all classes"... but I've never heard "relative" used in that way and I can't find anything except https://us.metamath.org/mpeuni/bj-disj2r.html

Would "restricted" be better here?

Reply:
I would use "relative". First, "restricted" could be understood as "special case", which is not the case here. Here, `|- -. { x e. A | -. x e. x } e. A ` is Russell's result *relative to A*. The relative point of view is useful for instance when doing stuff "relative to a universe" in set theory, or in a somewhat different way https://en.wikipedia.org/wiki/Grothendieck%27s_relative_point_of_view

By the way: I would prefer the formulation I wrote above, with `-.  X e.` instead of a mix of `-. X e.` and `X e/`.